client/src/analysis/components/FFTAnalysis.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { defaultDebounce } from "../../helpers";
+import ResultList from "../../job/components/ResultList";
+import { AnalysisTypes, DatasetOpen } from "../../messages";
+import { cbToRadius, inRectConstraint, keepOnCY, riConstraint, roConstraints } from "../../widgets/constraints";
+import Disk from "../../widgets/Disk";
+import DraggableHandle from "../../widgets/DraggableHandle";
+import Ring from "../../widgets/Ring";
+import { HandleRenderFunction } from "../../widgets/types";
+import * as analysisActions from "../actions";
+import { AnalysisState } from "../types";
+import AnalysisLayoutThreeCol from "./AnalysisLayoutThreeCol";
+import useFFTFrameView from "./FFTFrameView";
+import Toolbar from "./Toolbar";
+
+
+interface AnalysisProps {
```

Comment:
You can delete this interface and instead import the common `AnalysisProps` interface from types